packages/icons/src/createFondueIcon.ts

```diff
@@ -18,7 +18,7 @@ export type IconNode = [elementName: keyof ReactSVG, attrs: Record<string, strin
 export type SVGAttributes = Partial<SVGProps<SVGSVGElement>>;
 type ComponentAttributes = RefAttributes<SVGSVGElement> & SVGAttributes;
 
-type IconSizeAsNumber = 12 | 16 | 20 | 24 | 32;
+export type IconSizeAsNumber = 12 | 16 | 20 | 24 | 32;
```

Comment:
`export` might have been readded by mistake during a merge
```suggestion
type IconSizeAsNumber = 12 | 16 | 20 | 24 | 32;
```